scrapers/BelAmi.yml

```diff
@@ -0,0 +1,30 @@
+name: "BelAmi"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - belamionline.com
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: //*[@id="ContentPlaceHolder1_LabelTitle"]
+      Details:
+        selector: //*[@id="ContentPlaceHolder1_LabelDescription"]|//div[@class='bottom']/p[2]
+        concat: " "
+      Date:
+        selector: //*[@id="ContentPlaceHolder1_LabelReleased"]
+        postProcess:
+          - parseDate: 1/02/2006
```

Comment:
```yaml
          - parseDate: 1/2/2006
```
From a few random scenes it seems to be 1/2/2006, where there cases with 1/02/2006 ?

Reply:
Went back as far as 2012 to check the dates and all were using `1/2/2006` format. Confirmed and resolved.